fiaas_mast/models.py

```diff
@@ -6,7 +6,8 @@
     "application_name",
     "original_application_name",
     "spinnaker_tags",
-    "raw_tags"
+    "raw_tags",
+    "spinnaker_application"
```

Comment:
While this change is aligned with how the API has been extended up to now, I think that if we modify the API change here slightly, this can be made more orchestration layer agnostic, and more flexible in terms of changing the annotations used later. I'm thinking rather than add `spinnaker_application`  we could have something like `metadata_annotations`, and have the client send something like 
```json
{
  "metadata_annotations": {
    "moniker.spinnaker.io/application": "..."
  }
...
}
```
What do you think?

Reply:
+1